macros/parsers/parserMultiAnswer.pl

```diff
@@ -549,3 +421,163 @@ sub ans_array {
+are converted into C<MathObjects> if they aren't already.
+
+C<MultiAnswer> objects have the following attributes:
+
+=head2 checker
+
+A coderef to be called to check student answers. This is the only required attribute.
+
+The C<checker> routine is passed four parameters: a reference to the array of correct answers,
+a reference to the array of student answers, a reference to the C<MultiAnswer> object itself,
+and a reference to the checker's answer hash. The routine should return either a score or a
+reference to an array of scores (one for each answer).
+
+=head2 singleResult
+
+Indicates whether to show only one entry in the results table (C<< singleResult => 1 >>)
+or one for each answer rule (C<< singleResult => 0 >>). Default: 0.
+
+=head2 namedRules
+
+Indicates whether to use named rules or default rule names. Use named rules (C<< namedRules => 1 >>)
+if you need to intersperse other rules with the ones for the C<MultiAnswer>. In this case, you must
+use C<NAMED_ANS> instead of C<ANS>. Default: 0.
+
+=head2 checkTypes
+
+Specifies whether the types of the student and professor's answers must match exactly
+(C<< checkTypes => 1 >>) or just pass the usual type-match error checking (in which case, you should
+check the types before you use the data). Default: 1.
+
+=head2 allowBlankAnswers
+
+Indicates whether to remove the blank-check prefilter from the answer checkers used for type checking
+the student's answers. Default: 0.
+
+=head2 format
+
+An sprintf-style string used to format the students' answers for the results table when C<singleResult>
+is true. If undefined, the C<separator> parameter (below) is used to form the string. Default: undef.
+
+=head2 tex_format
+
+An sprintf-style string used to format the students' answer previews when C<singleResult> mode is
+in effect. If undefined, the C<tex_separator> (below) is used to form the string. Default: undef.
+
+=head2 separator
+
+The string to use between entries in the results table when C<singleResult> is set and C<format> is not.
+Default: semicolon.
+
+=head2 tex_separator
+
+The string to use as a separator between entries in the preview area when C<singleResult> is set
+and C<tex_format> is not. Default: semicolon followed by thinspace.
+
+=head1 METHODS
+
+=head2 setCmpFlags
+
+    $multianswer_obj->setCmpFlags($which_rule, %flags)
+
+Configure a specific comparison object within the C<MathObject> instance by setting various flags
+and their corresponding values.
+
+C<$which_rule> begins counting at 1.
+
+If the specified C<$which_rule> does not correspond to an existing comparison object within
+the C<MultiAnswer> instance, this method will throw an error with the message
+"Answer $which_rule is not defined."
+
+    $success = MultiAnswer($fraction_obj)->setCmpFlags(1, studentsMustReduceFractions => 1);
+    $failure = MultiAnswer($fraction_obj)->setCmpFlags(2, studentsMustReduceFractions => 1);
```

Comment:
This is a little confusing.  If you add code like this to the POD, authors may think that this might work.  I think you mean that the first example will throw an error message, and the second will not.  But that is not entirely clear.  Also, it seems to insinuate that the `$success` or `$failure` variables are something other than what they are.  The value of the `$success` variable would actually be the answer hash for the `$fraction_obj` answer.  Of course `$failure` would not be set because the `die` would occur.  In any case, the result of the `MultiAnswer` constructor call is never stored in a variable, and so the call would not be particularly useful.